fe1-web/src/core/components/QRCode.tsx

```diff
@@ -52,6 +55,13 @@ const QRCode = ({ value, visible, overlayText }: IPropTypes) => {
   // highest error correction if there is an overlay
   const errorCorrectionLevel = overlayText ? 'H' : 'L';
 
+  // Warns that the overlay is too big if the text is too small
+  if (overlayText && value && value.length < 59) {
+    console.warn(
+      'An overlay text has been added on a QRCode whose represents a too short text (length < 50)',
+    );
+  }
```

Comment:
What about forcing overlayText to false in this case and logging the warning / showing some text below the qr code that indicates the issue?

Reply:
The QR might still work. So I am not sure it is a good idea.

Reply:
It might but there is no guarantee. And I would certainly prioritize functionality over having a nice looking circle but this is up to you 😅